authorization/permission/model/permission_model_service.go

```diff
@@ -0,0 +1,179 @@
+package model
+
+import (
+	"context"
+
+	"github.com/fabric8-services/fabric8-auth/authorization/repository"
+	"github.com/jinzhu/gorm"
+	uuid "github.com/satori/go.uuid"
+)
+
+type PermissionModelService interface {
+	HasScope(ctx context.Context, identityID uuid.UUID, resourceID string, scope string) (bool, error)
```

Comment:
here `scope` is the scope name, right? do you think we could rename it to `scopeName` because it gets confusing at times if `scope` is the scopeID expressed a string.

Reply:
I've renamed it as suggested, good idea.